index.html

```diff
@@ -118,32 +112,8 @@ <h2>
         "<dfn>cleared</dfn>". Otherwise, it is said to be "<dfn>set</dfn>".
       </p>
       <p>
-        The application's badge MUST NOT contain the empty string, a negative
-        integer, or the integer value 0.
-      </p>
-      <div class="note">
-        <p>
-          A <a data-cite="UAX29#Grapheme_Cluster_Boundaries">grapheme
-          cluster</a> is usually a single Unicode character. However, it can
-          also contain multiple characters that are combined into what the user
-          thinks of as a single character. Examples of grapheme clusters
-          include:
-        </p>
-        <ul>
-          <li>"a" (a single character, Latin "a")
-          </li>
-          <li>"நி" (a base character + combining character, Tamil "ni")
-          </li>
-          <li>👩🏾‍🔬️ (emoji combination with skin tone modifier, Woman Scientist
-          with Medium-Dark Skin Tone)
-          </li>
-        </ul>
-      </div>
-      <p class="note">
-        Clients are encouraged to use integers instead of digit characters,
-        because some user agents may <em>only</em> support integer badges, so
-        using the digit character would not display correctly. Also integers
-        can be localized better for the user.
+        The application's badge MUST NOT contain a negative integer, or the
+        integer value 0.
```

Comment:
This is slightly unclear: it sounds like calling Badge.set(0) will cause an error, but what we're actually doing is special casing 0.

Reply:
This is the badge model section, not the interface. This section describes the value stored in the badge state (which cannot be 0), not how the `set` method works.

The `set` method special-cases 0 (as "set the application's badge to **nothing**"), but the model simply forbids the value 0.